src/tests/ftest/posix/simul.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import distro
+from avocado.core.exceptions import TestFail
+from dfuse_test_base import DfuseTestBase
+from env_modules import load_mpi
+from general_utils import DaosTestError
+from general_utils import run_command
+
+
+class PosixSimul(DfuseTestBase):
+    # pylint: disable=too-few-public-methods
```

Comment:
With 3 public methods, this should no longer be needed
```suggestion
```

Reply:
Will fix